eqcorrscan/tests/test_data/catalog.txt

```diff
@@ -0,0 +1,573 @@
+  47.708  32.737     9.1     0.0       7       5     3.0     166   0.176     0.6     0.6  2014 09 15 00 31 41.45
```

Comment:
Any chance we can use a shorter file, or a web-service to get this information? I have been guilty in the past of adding big/long files unnecessarily. The main issue is ideally we keep the repo small so that people can clone quickly, and keep the install small so it is fast to download. This is obviously quite a light file, but it would still be good not to have it if we don't need it.

Reply:
Sure, I wanted to show the result of this function with plotting a lot of points And make simple and quick test.
Because getting catalog from web-service is slower than this file.
I will explore to find and replace something better.

Reply:
Web-services can be slow sadly.  You will need to test passing a catalog to the function though as well as passing just locations, so you will need a full catalog at some point.